src/main/java/reposense/model/RepoConfiguration.java

```diff
@@ -219,9 +220,19 @@ public String getBranch() {
     }
 
     public void setBranch(String branch) {
+        if (this.branch.equals(CsvParser.DEFAULT_BRANCH)) {
```

Comment:
I previously did not have this check as well, I was specifically asked to include this check by @eugenepeh . Since both the reviews seem to be contradictory and asking me to undo changes specifically requested by previous reviewer, I would like you to have a look at the changes requested by the previous reviewer once before i proceed to make the changes.

Reply:
I believe you meant @eugenepeh comment [here](https://github.com/reposense/RepoSense/pull/332/files/c717ce69f54cff4088ad9a9afeffd3ce01316240#r221523249).
I believe what he meant was that the header comments does not correspond to what the method is doing (like my comment below), and to update the header comments, not to add the check.

Reply:
![image](https://user-images.githubusercontent.com/19277206/46740270-fd52df80-ccd4-11e8-9780-b43153de9d28.png)

I think the previous conversation goes like this? The check was already there before my review.

Reply:
Hmm, well this is just a slight nit anyway, can just ignore this comment.

Reply:
No, it doesn't change which is why i had to introduce updateDisplayName method.

Reply:
>No, it doesn't change which is why i had to introduce updateDisplayName method

What I meant is, even if the current branch name is not the default value, if it were to change to any other value, the display name should be updated as well (call the `updateDisplayName` method), so this check for default value shouldn't be in it right?.

@eugenepeh your take on this?

Reply:
Yeah, the check can be remove.

Reply:
Check has been removed